testing/btest/Baseline/core.tunnels.max-depth-exceeded/conn.log.cut

```diff
@@ -0,0 +1,5 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+uid	id.orig_h	id.resp_p	id.resp_h	id.resp_p	proto	history	service	tunnel_parents
+CHhAvVGS1DHFjwGM9	1.1.1.1	4789	1.1.1.9	4789	udp	D	vxlan	-
+ClEkJM2Vm5giqnMf4h	2.2.2.2	4789	2.2.2.9	4789	udp	D	vxlan	CHhAvVGS1DHFjwGM9
+C4J4Th3PJpwUYZZ6gc	3.3.3.3	4789	3.3.3.9	4789	udp	D?	-	ClEkJM2Vm5giqnMf4h
```

Comment:
I could also see `x` and `X`  to allow for directionality (more interesting if it's extended to other meanings). The `?` at the end of the history may also look confusing when asking questions.

What's the meaning of D??  - what is the meaning of `D?` ?

Reply:
My vote is for x/X